Vagrantfile

```diff
@@ -59,5 +59,7 @@ Vagrant.configure("2") do |box|
     # forward leap_web ports
     config.vm.network "forwarded_port", guest: 80,  host:8080
     config.vm.network "forwarded_port", guest: 443, host:4443
+    # forward pixelated ports
```

Comment:
the port 8080 used by leap does not work. But removing that should be another commit.
I can do that.
What do you mean by 'add a comment'?